DOC-11708 Deprecated REST API endpoint for bucket stats

supritha-kumar opened the pull request:
DOC-11708

For details about added contents, see [this comment](https://github.com/couchbase/docs-server/pull/3894#issue-3443514932).

supritha-kumar committed:
Update rest-buckets-table.adoc

supritha-kumar committed:
DOC-11708 Update rest-scopes-and-collections-table.adoc

replaced '/pools/default/buckets' with '/pools/defaults/stats/range'

supritha-kumar committed:
DOC-11708 Update rest-memory-and-storage-table.adoc

replaced pools/default/buckets with /pools/defaults/stats/range

supritha-kumar committed:
DOC-11708 Update rest-xdcr-table.adoc

replaced  /pools/default/buckets with /pools/defaults/stats/range

supritha-kumar committed:
/DOC-11708 Update rest-buckets-table.adoc

Updated post a conversation with Liam that only "/pools/default/buckets/[bucket-name]/stats" needs to change

supritha-kumar committed:
/DOC-11708 Update rest-scopes-and-collections-table.adoc

reverted earlier changes

supritha-kumar committed:
/DOC-11708 Update rest-memory-and-storage-table.adoc

reverted earlier changes

malarky commented on modules/rest-api/partials/rest-xdcr-table.adoc:
```diff
@@ -44,7 +44,7 @@
 
 | `GET`
-| `/pools/default/buckets/[source_bucket]/stats/[destination_endpoint]`
+| `//pools/defaults/stats/range/[source_bucket]/stats/[destination_endpoint]`
```
I'm not sure this is a drop-in replacement - you'll need to check with the XDCR team to see what's best here.

supritha-kumar replied:
Hi @malarky, have checked this with Liam, and resolving the conversation.

supritha-kumar committed:
Update rest-buckets-table.adoc

DOC-11708 
replaced GET /pools/default/buckets/[bucket-name]/stats with GET /pools/default/stats/range and added a link to the Statistics page.

Pallavi-Janardhan reviewed (approved):
LGTM

rao-shwe committed:
removed an extra slash in line 47

rao-shwe reviewed (approved):
Need to update original changes.

rao-shwe committed:
Corrected original content updates

rao-shwe committed:
Added xdcr rest api for getting statistics

rao-shwe committed:
updated important note

rao-shwe committed:
Edited important note

rao-shwe commented:
Note to @supritha-kumar and @rao-shwe:

1. 8.0 version of this content is reviewed by SMEs and fellow writers, and has been approved.
2. I won't merge this PR now as it will reflect in the public facing 7.6 Server docs immediately. So, let's merge this PR into the 7.6 branch during the 8.0 GA release.

rao-shwe commented:
**NOTE:**
Created the PR #3918 to revert merging. 
PR #3918 needs to be merged into 7.6 branch during the 8.0 GA release.
